### PR TITLE
Com.voxdsp.borg

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by BeverlyCode** [comment]
bot, build com.voxdsp.Borg

**Comment by BeverlyCode** [comment]
bot, build com.voxdsp.Borg

File: com.voxdsp.Borg.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+app-id: com.voxdsp.Borg
```
Comment (hfiguiere) [inline com.voxdsp.Borg.yaml:1]: Given what's at `voxdp.org` I have doubt this is related. So please use an app-id that meet the requirements.

Comment (BeverlyCode) [reply com.voxdsp.Borg.yaml:1]: this is related, it's a domain I control it's just a holding page. What would you suggest otherwise?

**Comment by BeverlyCode** [comment]
bot, build com.voxdsp.Borg

**Comment by BeverlyCode** [comment]
bot, build com.voxdsp.Borg

File: com.voxdsp.Borg.yaml (hunk 1 above)
Comment (hfiguiere) [reply com.voxdsp.Borg.yaml:1]: ok then.

File: com.voxdsp.Borg.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    buildsystem: simple
+    build-commands:
+      - cc main.c -Ofast -lSDL2 -lm -o borg
+      - install -Dm 0755 borg -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/borg.desktop ${FLATPAK_DEST}/share/applications/borg.desktop
+      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.appdata.xml
+      - install -Dm 0644 flatpak/borg.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/borg.png
+    sources:
+      - type: git
+        url: https://github.com/BeverlyCode/Borg-ER-3
```
Comment (barthalion) [inline com.voxdsp.Borg.yaml:26]: Do you plan to maintain it as a full-fledged fork? I noticed it's been forked from https://github.com/mrbid/Borg-ER-3.

Comment (BeverlyCode) [reply com.voxdsp.Borg.yaml:26]: Honestly, it's my repository and project, I am mrbid, so yeah I will maintain this as a full-fledged fork.

Comment (mrbid) [reply com.voxdsp.Borg.yaml:26]: verified

Comment (mrbid) [reply com.voxdsp.Borg.yaml:26]: > Honestly, it's my repository and project, I am mrbid, I will maintain this as a full-fledged fork.
> 
> Same for #3834

I can verify this statement.

Comment (BeverlyCode) [reply com.voxdsp.Borg.yaml:26]: I will maintain this as a full-fledged fork.

Same for https://github.com/flathub/flathub/pull/3834

Comment (mrbid) [reply com.voxdsp.Borg.yaml:26]: > I will maintain this as a full-fledged fork.
> 
> Same for #3834

I can verify this statement.

**Comment by BeverlyCode** [comment]
bot, build com.voxdsp.Borg

**Comment by BeverlyCode** [comment]
do these update automatically?

**Pull request closed without merging**